src/libpgmoneta/restore.c

```diff
@@ -153,11 +172,28 @@ reconstruct_backup_file(int server,
                         char* relative_dir,
                         char* bare_file_name,
                         struct deque* prior_labels,
-                        struct art* prior_backups,
+                        struct art* backups,
                         bool incremental,
                         int algorithm,
                         struct json* files);
 
+static void
+do_reconstruct_backup_file(struct worker_common* wc);
+
+static void
+create_reconstruct_backup_file_input(int server,
```

Comment:
Should we wait for _load / _save ?

Reply:
This is just for creating input for workers, not backup.info related... We can do another round with the naming and stuff though.